build/gen_stub.php

```diff
@@ -2226,11 +2265,13 @@ public function getDeclaration(iterable $allConstInfos): string {
         $code .= "\tzend_string *property_{$propertyName}_name = zend_string_init(\"$propertyName\", sizeof(\"$propertyName\") - 1, 1);\n";
         $nameCode = "property_{$propertyName}_name";
 
-        if ($this->type !== null) {
-            $template = "\tzend_declare_typed_property(class_entry, $nameCode, &$zvalName, %s, NULL, $typeCode);\n";
+        if (!empty($this->attributes)) {
+            $template = "\tzend_property_info *property_" . $this->name->getDeclarationName() . " = ";
         } else {
-            $template = "\tzend_declare_property_ex(class_entry, $nameCode, &$zvalName, %s, NULL);\n";
```

Comment:
It was necessary to change to `zend_declare_typed_property`, because unfortunately `tzend_declare_property_ex` doesn't return the `zend_property_info`.